stigs/system/ksp-block-stig-rhel-v-230336.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-stig-rhel-v-230336
+  namespace: default # Change your namespace
+spec:
+  tags: ["STIG","RHEL"]
+  message: "Alert! Access to this files will be Audit"
```

Comment:
@nandakrr the message is unclear. Please give a proper message so that user can understand what happened
cc: @salman-accuknox @raviknox